woocommerce/compatibility/class-sv-wc-order-compatibility.php

```diff
@@ -522,6 +524,64 @@ public static function get_edit_order_url( \WC_Order $order ) {
 	}
 
 
+	/**
+	 * Returns the admin screen id for orders.
+	 *
+	 * This method detects the screen_id according to HPOS availability,
+	 * as cannot rely anymore on the shop_order registered post type as the screen ID.
+	 *
+	 * @since x.y.z
+	 *
+	 * @return string
+	 */
+	public static function get_screen_id( ) {
+
+		return SV_WC_Plugin_Compatibility::is_hpos_enabled() ? wc_get_page_screen_id( 'shop-order' ) : 'shop_order';
+	}
+
+
+	/**
+	 * Determines whether the given $post_id is a WooCommerce order or not according to HPOS availability.
+	 *
+	 * @since x.y.z
+	 *
+	 * @param int $post_id post ID.
+	 * @return bool
+	 */
+	public static function is_order( $post_id ) {
```

Comment:
```suggestion
	public static function is_order_screen( int $post_id ) : bool {
```

I recommend changing this to `is_order_screen` to avoid confusions with the order object